bootstrap/Helper.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import requests
+
+
+def setup_logger(name):
+    """
+    :param name: name
+    :return: logger
+    """
+    # create logger
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.DEBUG)
+
+    # create console handler and set level to debug
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.DEBUG)
+
+    # create formatter
+    formatter = logging.Formatter('%(asctime)s %(levelname)s %(name)s: %(message)s')
+
+    # add formatter to ch
+    ch.setFormatter(formatter)
+
+    # add ch to logger
+    logger.addHandler(ch)
+
+    return logger
+
```

Comment:
Can you print an example of what a debug line would have in terms of date time stamp?
Here is what is used in New Relic: `2019-05-04T00:04:03.117Z: Step 4: 3.5s: enterUserName`
This datetime stamp exactly matches that which is found in splunk logs so easy chronological forensics can be executed.